### PR TITLE
Change the query API to match the new changes made to the matrix API

### PR DESCRIPTION
**Pull request by dkhaldi**
https://github.com/intel/llvm/pull/6662

### CODE REVIEW

**Comment by dkhaldi** [comment]
ping @yubingex007-a11y

File: sycl/include/sycl/ext/oneapi/matrix/static-query-use.hpp [hunk 1]
```diff
@@ -363,7 +363,7 @@ struct tpu_params<tpu::dpas, Ta, Tb, Tc, 0, 0, 0,
     uint32_t max_ksize;
     matrix_type atype;
     matrix_type btype;
-    matrix_type ctype;
+    matrix_type acuumulatortype;
```
Comment (yubingex007-a11y) [inline sycl/include/sycl/ext/oneapi/matrix/static-query-use.hpp:366]: acumulatortype

File: sycl/test/matrix/query-use.cpp [hunk 2]
```diff
@@ -0,0 +1,143 @@
+// RUN: %clangxx -DSYCL_EXT_ONEAPI_MATRIX_VERSION=2 -fsycl -o query-use %s
```
Comment (yubingex007-a11y) [inline sycl/test/matrix/query-use.cpp:1]: do we have a testcase in llvm-test-suite and check the query result?

Comment (dkhaldi) [reply sycl/test/matrix/query-use.cpp:1]: > do we have a testcase in llvm-test-suite and check the query result?

https://github.com/intel/llvm-test-suite/pull/1358 adds the test

Comment (dkhaldi) [reply sycl/test/matrix/query-use.cpp:1]: Ping @yubingex007-a11y

**Review by yubingex007-a11y — APPROVED** [review]

**Comment by dkhaldi** [comment]
@intel/llvm-reviewers-runtime, can you help review/merge?

**Comment by dkhaldi** [comment]
@intel/llvm-reviewers-runtime, can you please help merge this?

**Comment by dkhaldi** [comment]
ping https://github.com/orgs/intel/teams/llvm-reviewers-runtime

**Review by againull — APPROVED** [review]

**Pull request merged**